### PR TITLE
Bump to 3.36.1

### PR DESCRIPTION
**Pull request by tschoonj**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,35 +11,37 @@ package:
...
-    - {{posix}}patch
-    - {{native}}pkg-config
+    - {{  posix  }}patch
+    - {{  native  }}pkg-config
   host:
     - flake8  # for testing during the build
     - glib
-    - gobject-introspection 1.*
+    - gobject-introspection
+    - gtk3  # for testing during the build
```
Comment (pkgw) [inline recipe/meta.yaml:36]: I believe that, by default, adding this dep will propagate a runtime dependency on `gtk3` into the generated packages. We don't want that to occur, and we should be careful to check that it doesn't happen.

**Comment by tschoonj** [comment]
I got it working now, but I had to disable the tests for it to work, as they were either breaking compilation or didn't run without errors. The conda-forge tests work fine though, and demonstrate that it actually works.

If you agree, I will clean things up and squash my commits.

I also got rid of gtk3, which appeared to be required to run for some of its tests.

**Comment by pkgw** [comment]
@tschoonj Do you think you could get it so that the test suite passes on at least one platform, maybe Linux/x86_64? The test suites for packages like this often aren't reliable across all platforms, but I'd feel a lot better if we at least maintained the ability to run the test suite *somewhere*.

Clearly you've spent time on this, though, and if your takeaway is that it will be prohibitive to get the test suite running anywhere, then I'd rather get the version bump than keep the test suite.

**Comment by tschoonj** [comment]
I honestly wouldn't bother trying to run `ninja test` for any platform: even for Linux some tests already need to be disabled for this to run, plus I need to add a hack to copy files from gobject-introspection, just to avoid a `ninja: file path is too long` error.

**Pull request merged**